demo/src/features/text/jobs/write.rs

```diff
@@ -148,17 +148,6 @@ impl<'write> RenderFeatureWriteJob<'write> for TextWriteJob<'write> {
                         )],
                     )?;
             }
-
-            begin_execute_graph_context
```

Comment:
Why did this get removed?

Reply:
the lines just above already put the resource into the SHADER_RESOURCE state, so this transition is incorrect and should be removed